terraform/infrastructure/etc/prod.tfvars

```diff
@@ -1,4 +1,5 @@
-account_name        = "prod"
-domain              = "api.record-locator.national.nhs.uk"
-public_domain       = "api.service.nhs.uk"
-deletion_protection = true
+account_name         = "prod"
+domain               = "api.record-locator.national.nhs.uk"
+public_domain        = "api.service.nhs.uk"
+deletion_protection  = true
+log_retention_period = 2192
```

Comment:
I think this is 6 years + 2 days, is that to account for leap years? :brain:

Reply:
yeah, cloudwatch has certain numbers as well you can choose from too